internal/kms/kms.go

```diff
@@ -315,6 +317,46 @@ func (k *Kms) ReconcileKeys(ctx context.Context, randomReader io.Reader) error {
 			errors.Wrap(ctx, err, op)
 		}
 	}
+
+	opts := getOpts(opt...)
+	for _, id := range opts.withScopeIds {
+		var scopeRootWrapper *multi.PooledWrapper
+
+		for _, purpose := range []KeyPurpose{
+			// just add additional purposes as needed going forward to reconcile
+			// new keys as they are added.
+			//
+			// NOTE: don't add an audit key here since it can only be created in
+			// the global scope.
+			KeyPurposeOidc,
```

Comment:
Are we already creating OIDC DEKs for non-global/org scopes? If not we may want a check here since this will be called with _all_ scopes from the controller New function.

Reply:
We are calling `kms.CreateKeysTx(...)` every time we create a new scope.